scripts/generate-test-services.ts

```diff
@@ -85,12 +85,14 @@ async function generateTestServicesWithLocalCoreModules(
   }
 
   (await readServiceDirectories()).forEach(serviceDirectory =>
-    readServiceDirectory(serviceDirectory).then(files =>
-      files.forEach(file =>
-        readServiceFile(serviceDirectory, file).then(data => {
-          replaceWithLocalModules(serviceDirectory, file, data);
-        })
-      )
+    readServiceDirectory(serviceDirectory).then(dirents =>
+      dirents
```

Comment:
`dirents` are either files or directories? I do not really like the name. Perhaps `serviceDirectoryContent`?

Reply:
Added the `fs.Direent` type name for less confusing.